resources/views/user/details.blade.php

```diff
@@ -0,0 +1,334 @@
+								<p id="courseSchedule"></p>
+							</div>
+							<div class="actions">
+								<button type="button" class="ui cancel button">Cancel</button>
+								<button type="button" id="confirmAddCourse" class="ui primary button" onclick="associateCourse()">Confirm</button>
+							</div>
+						</div>
+						<div id="courseAddedSuccessfully" class="ui coupled second modal">
+							<div class="header">Course successfully added</div>
+							<div class="content">
+								<i class="check huge green circle icon"></i>
+								<p>You have been added to</p>
+								<p id="courseAddedName"></p>
+							</div>
+							<div class="actions">
+								<button type="button" class="ui ok primary button">Done</button>
+							</div>
+						</div>
+						<div id="courseNotFound" class="ui modal">
+							<div class="header">Course not found</div>
+							<div class="content">
+								<i class="times huge red circle icon"></i>
+								<p>The course key you entered was not found.</p>
+							</div>
+							<div class="actions">
+								<button type="button" class="ui ok primary button">Done</button>
+							</div>
+						</div>
+						<div id="courseDuplicated" class="ui modal">
+							<div class="header">Duplicated course</div>
+							<div class="content">
+								<i class="times huge red circle icon"></i>
+								<p>You are already associated with this course.</p>
+							</div>
+							<div class="actions">
+								<button type="button" class="ui ok primary button">Done</button type="button">
+							</div>
+						</div>
+						<div id="courseAdditionError" class="ui modal">
+							<div class="header">Something went wrong</div>
+							<div class="content">
+								<i class="times huge red circle icon"></i>
+								<p>We were unable to associate you with this course.</p>
+							</div>
+							<div class="actions">
+								<button type="button" class="ui ok primary button">Done</button type="button">
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+@section("scripts")
+<script>
+	/* Semantic UI setup */
+	$('.menu .item')
+		.tab()
+	;
+
+	$(".coupled.modal").modal({
+			allowMultiple: false
+	});
+
+	/* send cstf token on every ajax request */
+	$.ajaxSetup({
+		beforeSend: function(xhr) {
+			xhr.setRequestHeader('X-CSRF-TOKEN', $('meta[name="csrf-token"]').attr("content"));
+		}
+	});
+
+	/* courses */
+	let hasCourses = {!! $user->enrolledIn !!}.length > 0;
+
+	function showCourses() {
+		if(hasCourses) {
+			$('#current-courses').show();
+			$('#no-courses-msg').hide();
+
+		}
+		else {
+			$('#current-courses').hide();
+			$('#no-courses-msg').show();
+		}
+	}
+
+	showCourses(); // execute on load
+
+	@if($user->id == Auth::user()->id)
+		/* course association workflow */
+		function validateCourseKey(courseKey) {
+			$("#courseKeyInputContainer").removeClass("error");
+			$("#courseKey").removeClass("error");
+
+			if(courseKey == "") {
+				$("#courseKeyInputContainer").addClass("error");
```

Comment:
```suggestion
				$("#courseKeyInputContainer", "#courseKey").addClass("error");
```

Y borras la que sigue